static/locales/ja.json

```
@@ -109,12 +109,18 @@
         "live2dPreviewLoadFailed": "Live2Dモデルプレビューの読み込みに失敗しました: {{error}} ",
-            "vrmModuleLoadFailed": "VRM モジュールの読み込みに失敗しました",
-            "vrmPreviewLoaded": "VRM モデルのプレビューを読み込みました",
-            "vrmPreviewFailed": "VRM モデルのプレビュー読み込みに失敗しました",
-            "mmdModuleLoadFailed": "MMD モジュールの読み込みに失敗しました",
-            "mmdPreviewLoaded": "MMD モデルのプレビューを読み込みました",
-            "mmdPreviewFailed": "MMD モデルのプレビュー読み込みに失敗しました",
+        "coverBgBottom": "背景下層",
+        "characterCardCover": "キャラクターカードのカバー",
+        "coverBgTop": "背景上層",
+        "selectCharacterForCover": "カバーを表示するキャラクターを選択してください",
+        "searchAriaLabel": "サブスクリプションを検索",
```

Biome (2.4.10)

[error] The key searchAriaLabel was already declared.

(lint/suspicious/noDuplicateObjectKeys)